sdk/digitaltwins/digitaltwins/test/README.md

```diff
@@ -0,0 +1,5 @@
+# Testing
+
+To test this project, make sure to build it by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building), then follow the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The DigitalTwins client does not have any recorded tests and so, all the tests require an Azure Cognitive Search account to be set up beforehand. You can use existing Azure resources for the live tests.
```

Comment:
`Cognitive Search` ? :)

Also I think the plan is to have recorded tests eventually?

Reply:
Fixed, and yes, the plan is to have full recorder coverage.

---
In reply to: [470205842](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470205842) [](ancestors = 470205842)